src/cli/cli.cc

```diff
@@ -2706,6 +2706,17 @@ int main (const int argc, const char* argv[]) {
       fs::create_directories(paths.pathBin);
       fs::create_directories(pathResources);
 
+      if (settings.contains("mac_info_plist_file")) {
```

Comment:
are this one and `ios_info_plist_file` a new config parameters?

Reply:
yes, undocumented for right now 🙏🏼